tests/test_app.py

```diff
@@ -1,4 +1,228 @@
+    test_list.append(test_item)
+    return test_list      
+
+@pytest.fixture
+def board_done_four_today_one_yesterday():
+    today=datetime.now()
+    yesterday=today-timedelta(days=1)
+    today_str = datetime.strftime(today,'%Y-%m-%dT%H:%M:%S.%fZ')
+    yesterday_str = datetime.strftime(yesterday,'%Y-%m-%dT%H:%M:%S.%fZ')
+    test_list=[]
+    i = 1
+    test_item = Item(name='Dick',status='Done',dateLastActivity=today_str)
+    while i <= 4:
+        test_list.append(test_item)
+        i += 1
+    test_item = Item(name='Harry',status='Done',dateLastActivity=yesterday_str)
+    test_list.append(test_item)
+    return test_list      
+
+@pytest.fixture
+def board_four_in_done_today():
+    today = datetime.now()
+    today_str = datetime.strftime(today,'%Y-%m-%dT%H:%M:%S.%fZ')
+    test_list=[]
+    i = 1
+    test_item = Item(name='Harry',status='Done',dateLastActivity=today_str)
+    while i <= 4:
+        test_list.append(test_item)
+        i += 1
+    return test_list      
+
+@pytest.fixture
+def board_five_in_done_today():
+    today = datetime.now()
+    today_str = datetime.strftime(today,'%Y-%m-%dT%H:%M:%S.%fZ')
+    test_list=[]
+    i = 1
+    test_item = Item(name='Harry',status='Done',dateLastActivity=today_str)
+    while i <= 5:
+        test_list.append(test_item)
+        i += 1
+    return test_list   
+
+@pytest.fixture
+def board_done_1today_2yesterday_3last_week_4tomorrow():
+    test_list=[]
+    today = datetime.now()
+    tomorrow = today + timedelta(days=1)
+    yesterday = today - timedelta(days=1)
+    lastweek = today - timedelta(days=7)
+    today_str = datetime.strftime(today,'%Y-%m-%dT%H:%M:%S.%fZ')
+    yesterday_str = datetime.strftime(yesterday,'%Y-%m-%dT%H:%M:%S.%fZ')
+    lastweek_str = datetime.strftime(lastweek,'%Y-%m-%dT%H:%M:%S.%fZ')
+    tomorrow_str = datetime.strftime(tomorrow,'%Y-%m-%dT%H:%M:%S.%fZ')
+
+    test_item = Item(name='Tom is done today', dateLastActivity = today_str, status='Done')
+    test_list.append(test_item)
+    test_item = Item(name='Dick is done yesterday', dateLastActivity = yesterday_str, status='Done')
+    test_list.append(test_item)
+    test_list.append(test_item)
+    test_item = Item(name='Harry is done last week',dateLastActivity = lastweek_str, status='Done')
+    test_list.append(test_item)
+    test_list.append(test_item)
+    test_list.append(test_item)
+    test_item = Item(name='Nobody can be done tomorrow',dateLastActivity = tomorrow_str, status='Done')
+    test_list.append(test_item)
+    test_list.append(test_item)
+    test_list.append(test_item)
+    test_list.append(test_item)
+
+    return test_list    
+
+@pytest.fixture
+def board_one_in_each():
+    tom_item = Item(name='Tom',status='To Do')
+    dick_item = Item(name='Dick',status='Doing')
+    harry_item = Item(name='Harry',status='Done')
+    test_list=[]
+    test_list.append(tom_item)
+    test_list.append(dick_item)
+    test_list.append(harry_item)
+    return test_list    
```

Comment:
Minor, but this might be easier to capture in the condensed form where we create & add each Item at the same time:
```suggestion
    test_list = [
        Item(id=1, name='Tom',status='To Do'),
        Item(id=1, name='Dick',status='Doing'),
        Item(id=1, name='Harry',status='Done')
    ]
    return test_list
```